screwdriver.yaml

```diff
@@ -68,5 +68,9 @@ jobs:
             K8S_DEPLOYMENT: sdapi
             SD_API: api.screwdriver.cd
         secrets:
+            # Access key for functional tests
+            - ACCESS_KEY
```

Comment:
I think this should be `SECRET_ACCESS_KEY`, but https://github.com/screwdriver-cd/screwdriver/pull/243 should fix it.